testsuite/tests/lib-array/test_array.ml

```diff
@@ -177,3 +177,10 @@ let a : int array =
 [%%expect{|
 val a : int array = [|2; 4; 6; 8|]
 |}]
+
+let n : int =
+  let a = [| 1 ; 2 ; 3 ; 4 |] in
+  Array.count_if (fun n -> n mod 2 = 0) a;
```

Comment:
this should be `;;` or nothing, I suspect that `;` may create parsing errors if we add more sentences later. (Or maybe not thanks to the structure extension that follows, but still, fragile style.)